src/Controller.cpp

```diff
@@ -122,6 +141,15 @@ void Controller::indicateMode(byte mode)
     _ui->indicateMode(mode);
   }
   
+  if ((mode == MODE_UNINITIALISED) || (mode == MODE_SETUP))
+  {
+    _mparams[PAR_FLAGS] &= ~PF_FLiM;
```

Comment:
Sorry for giving you the wrong code example. Of course there has to be a ~ here.